content/doc/book/updating/servlet-containers.adoc

```diff
@@ -0,0 +1,43 @@
+---
+layout: section
+title: Other Servlet Containers
+---
+ifdef::backend-html5[]
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:toc: left
+:toclevels: 4
+:hide-uri-scheme:
+ifdef::env-github[:imagesdir: ../resources]
+ifndef::env-github[:imagesdir: ../../resources]
+endif::[]
+
+== Servlet container
+
+include::doc/book/updating/_pre-upgrade-wizard.adoc[]
+
+Once a .war archive has been downloaded, follow the servlet container’s existing application deployment process.
+
+When using servlet containers, Jenkins will set the `JENKINS_HOME` to the `$APP_SERVER_USER/.jenkins/` folder. 
+If the servlet container installation *does not include write permissions* to this folder for this user (sometimes done for security), you either need to grant appropriate permissions or override 
+this setting by adding the `-DJENKINS_HOME=$MY_JENKINSPATH` argument in your servlet container startup. 
+Refer to the servlet container’s documentation for how to add startup arguments.
+
+== Tomcat
+Use environment variable CATALINA_OPTS to add:
+[source]
+```
+-Dorg.apache.tomcat.util.buf.UDecoder.ALLOW_ENCODED_SLASH=true.
+```
+[source]
+```
+-Dorg.apache.catalina.connector.CoyoteAdapter.ALLOW_BACKSLASH=true
+```
+which is needed for Blue Ocean.
+
+It is recommended to configure it in the script `$CATALINA_BASE/bin/setenv.sh (Linux) or `%CATALINA_BASE%\bin\setenv.bat` (Microsoft Windows) that you’ll create to customize your application server.
```

Comment:
```suggestion
It is recommended to configure it in the script you’ll create to customize your application server as `$CATALINA_BASE/bin/setenv.sh` (Linux) or `%CATALINA_BASE%\bin\setenv.bat` (Windows).
```
small update for message flow, if this removes the original intention, disregard